text/0044-heartbeat.md

```diff
@@ -0,0 +1,69 @@
+- Start Date: 2022-12-09
+- RFC Type: feature
+- RFC PR: N/A
+- RFC Status: draft
+
+# Summary
+
+Introduce an infrastructure monitoring agent in all SDKs supported by us. This agent will be triggered every time the user launches the application and will send us information about the state of their integration with Sentry.
```

Comment:
I would not call this "infrastructure monitpring agent". This has a different meaning in the industry. What we want is a heartbeat that carries diagnostic data.

Reply:
will update this...thanks for the feedback